client/my-sites/upgrades/domain-management/email/index.jsx

```diff
@@ -68,7 +69,7 @@ const Email = React.createClass( {
 
 	content() {
 		if ( ! ( this.props.domains.hasLoadedFromServer && this.props.googleAppsUsersLoaded ) ) {
-			return this.translate( 'Loading…' );
+			return this.loadingPlaceholder();
```

Comment:
I was just following the pattern I've noticed here - no actual components in `content()`, everything is delegated to another render function. I know it's an overkill for the loading placeholder and I can change it to inline if I have indeed overdone it :wink: